ui/component/searchTopClaim/view.jsx

```diff
@@ -14,21 +14,28 @@ type Props = {
   winningUri: ?Claim,
   doResolveUris: (Array<string>) => void,
   hideLink?: boolean,
+  setChannelActive: boolean => void,
 };
 
 export default function SearchTopClaim(props: Props) {
-  const { doResolveUris, query = '', winningUri, hideLink = false } = props;
+  const { doResolveUris, query = '', winningUri, hideLink = false, setChannelActive } = props;
   const uriFromQuery = `lbry://${query}`;
 
   let channelUriFromQuery;
+  let winningUriIsChannel;
   try {
     const { isChannel } = parseURI(uriFromQuery);
-
+    const { isChannel: winnerIsChannel } = parseURI(winningUri);
+    winningUriIsChannel = winnerIsChannel;
```

Comment:
This can just use `isChannel`. It doesn't need to call `parseURI` twice

Reply:
I'll take another look - one is from the query, the other from the return of winningUri.